ejercicios/src/ar/edu/ub/pcsw2016/diegovega/Diego.java

```diff
@@ -3,18 +3,54 @@
 /**
  * Created by Pulpo on 06/08/2016.
  */
-abstract public class Diego implements IExercise, IStudent(String name, Integer registrationNumber){
-    private IExercise Ex;
-    private IStudent St;
+public class Diego implements IStudent{
     private String name;
-    private Integer registrationNumber;
-    private List<String> excersice = new ArrayList<String>();
+    private int registrationNumber;
+    private List<IExercise> excerciseList = new ArrayList<IExercise>();
 
-        public void addExcersice(IExercise exercise){
-            excersice.add(excersice);
+    public int getRegistrationNumber() {
+        return registrationNumber;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+
+    public Diego(String name, int registrationNumber){
+        this.name = name;
+        this.registrationNumber = registrationNumber;
```

Comment:
Ah sisi lo pens? eso je. A la tarde lo cambio. Gracias!

Sent from my iPhone

On Aug 8, 2016, at 09:43, Didier Renard <notifications@github.com<mailto:notifications@github.com>> wrote:


In ejercicios/src/ar/edu/ub/pcsw2016/diegovega/Diego.java<https://github.com/didierrenardub/ub_pcsw_2016/pull/2#discussion_r73868153>:

>
> -        public void addExcersice(IExercise exercise){
> -            excersice.add(excersice);
> +    public int getRegistrationNumber() {
> +        return registrationNumber;
> +    }
> +
> +    public String getName() {
> +        return name;
> +    }
> +
> +
> +    public Diego(String name, int registrationNumber){
> +        this.name = name;
> +        this.registrationNumber = registrationNumber;


Ac? directamente pod?s hacer que el constructor no reciba par?metros, e inicializar name y registrationNumber con tus datos; se supone que no van a cambiar con el tiempo.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/didierrenardub/ub_pcsw_2016/pull/2/files/51788e70a225530a561cadebca1938ea26259249..5c3bd941ae5d05732dde33809a665e9b90af953d#r73868153>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AHonux6wr1QSEwjXXnRLAyhb76qMTGS6ks5qdyRwgaJpZM4Jel3t>.